package.json

```diff
@@ -42,6 +42,7 @@
     "react": "^16.8.0"
   },
   "dependencies": {
+    "dtslint": "^2.0.5",
```

Comment:
What's the issue with the pre-commit check that needs this to be added as a dependency (not devdependency?)?